hw2/04-background.js

```diff
@@ -1 +1,41 @@
 // Add your code here
+let input = document.getElementById("userinput");
+const background = document.querySelector("body");
+let button = document.getElementById("button");
+button.addEventListener("click", changColor);
+
+let interval;
+window.addEventListener("load", () => {
+  interval = setInterval(function () {
+    background.style.backgroundColor = color();
+  }, 3000);
+  button.value = "Stop";
+  button.style.background = "red";
```

Comment:
Instead of changing the color directly, you can switch between the `btn-primary` and `btn-danger` Bootstrap CSS classes.

https://getbootstrap.com/docs/5.0/components/buttons/

Reply:
good suggestion! :)